crypto/fipsmodule/kdf/internal.h

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <openssl/digest.h>
+#include <openssl/hmac.h>
+
+typedef struct {
+  void *data;
+} SSKDF_VARIANT_CTX;
+
+typedef struct {
+  const EVP_MD *digest;
+  EVP_MD_CTX *ctx;
```

Comment:
nit: `md_ctx` instead of `ctx` would help with readability of the code where this is being used.